pkg/apis/testharness/v1beta1/test_types.go

```diff
@@ -107,6 +107,8 @@ type TestStep struct {
 type TestAssert struct {
 	// The type meta object, should always be a GVK of  kuttl.dev/v1beta1/TestAssert or kuttl.dev/v1beta1/TestAssert.
 	metav1.TypeMeta `json:",inline"`
+	// Override the default metadata. Set labels or override the test step name.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
while we use the k8s object structure... do don't actually use it to create objects in a cluster... I can't see that this is used.. is there value or a necessity to add it?  Not opposed.. more curious